client/components/main/header/Header.jsx

```diff
@@ -35,6 +35,11 @@ const Header = () => {
 
       <div id="navbar" className="navbar-menu">
         <div className="navbar-end">
+          <div className="navbar-item">
```

Comment:
Could you move the help center link to the right side of the header?

Reply:
Move to the right! Combined with the responsive navbar you pushed up, it now pops up on the left next to toggle when page scales down FYI